--- setup.py.orig	2006-02-01 17:47:44.000000000 +0000
+++ setup.py	2006-02-01 17:48:01.000000000 +0000
@@ -152,7 +152,7 @@
         f.write('import os\n')
         f.write('VERSION=\'%s\'\n' % VERSION)
         # expand backspaces
-        f.write('DATA_DIR=\'%s\'\n' % data_dir.replace('\\', '\\\\'))
+        f.write('DATA_DIR=\'/usr/share/gaphor\'\n')
         if os.name == 'nt':
             home = 'USERPROFILE'
         else:
